apps/api/Dockerfile

```diff
@@ -58,6 +61,8 @@ RUN --mount=type=cache,id=pnpm-store-api,target=/root/.pnpm-store\
  --unsafe-perm \
  --reporter=silent
 
+RUN if $BULL_MQ_PRO_NPM_TOKEN ; then rm -f .npmrc ; fi
```

Comment:
I think we can leave this one here, any particular reason to remove it?

Reply:
I did read this article and was thinking of if we are going to offer this to third parties they recommend doing it...
https://www.alexandraulsh.com/2018/06/25/docker-npmrc-security/

Reply:
This should be removed from the container as it could expose our tokens for bullmq pro.